trunner/target/armv7a9.py

```diff
@@ -19,26 +19,59 @@
 )
 from trunner.harness import TerminalHarness
 from trunner.host import Host
-from trunner.tools import Phoenixd, OpenocdGdbServer, wait_for_vid_pid
+from trunner.tools import Phoenixd, OpenocdGdbServer, Openocd, wait_for_vid_pid
 from trunner.types import TestResult, TestOptions
 from .base import TargetBase, find_port
 
 
 class ARMv7A9TargetRebooter(Rebooter):
     # TODO add text mode reboot
 
+    def __call__(self, flash=False, hard=False):
+        """Sets flash mode and perform hard or soft & debugger reboot based on `hard` flag."""
+
+        self._set_flash_mode(flash)
+
+        if hard:
+            if self.host.has_gpio():
+                self._reboot_dut_gpio(hard=hard)
+            else:
+                self._reboot_by_debugger()
+        else:
+            self._reboot_by_debugger()
```

Comment:
What about the` _reboot_dut_text() `?

I know we don’t have it implemented yet, but we want to have it